extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -388,15 +388,24 @@ export class ProjectsController {
 		const label = codeAnalysis
 			? constants.buildWithCodeAnalysisTaskName
 			: constants.buildTaskName;
-		const command = codeAnalysis
-			? `${constants.dotnetBuild} ${project.projectFilePath} ${constants.runCodeAnalysisParam} ${buildArguments}`
-			: `${constants.dotnetBuild} ${project.projectFilePath} ${buildArguments}`;
+
+		// Create an array of arguments instead of a single command string
+		const args: string[] = [constants.build, project.projectFilePath];
+
+		if (codeAnalysis) {
+			args.push(constants.runCodeAnalysisParam);
+		}
+
+		// Parse buildArguments and add them as separate array elements
+		const additionalArgs = buildArguments.trim().split(/\s+/);
```

Comment:
Where does buildArguments come from? If a user can set this it can have spaces. For example, this is a valid command:
`dotnet build -p:SetVar="My Variable" -o "./Custom Output Path/"`

Reply:
We prepare them in buildhelper file which I have modified and all args will be handled if they have any spaces.